src/js/events/MarathonActions.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
       error: function (xhr) {
         AppDispatcher.handleServerAction({
           type: REQUEST_MARATHON_SERVICE_CREATE_ERROR,
-          data: RequestUtil.getErrorFromXHR(xhr),
+          data: JSON.parse(xhr.response),
```

Comment:
We have `RequestUtil. parseResponseBody` I believe you could use here: https://github.com/dcos/mesosphere-shared-reactjs/blob/master/utils/RequestUtil.js#L112